__tests__/app.test.js

```diff
@@ -213,6 +213,54 @@ describe('PATCH commands', () => {
   
 });
 
+describe('Get /api/reviews Queries', () => {
+    test('200: should return the reviews with the category dexterity', () => {
+        return request(app).get("/api/reviews?category=dexterity")
+        .expect(200)
+        .then(({body})=>{
+            
+            const reviews = body.reviews;
+          
+            expect(reviews).toHaveLength(1);
+            reviews.forEach((review)=>{
+                expect(review).toHaveProperty("category")
+            })            
+            })
+        });
+
+    test('200: should return no reviews with category children\'s games', () => {
+        return request(app).get("/api/reviews?category=children's+games")
+        .expect(200)
+        .then(({body})=>{
+            const reviews = body.reviews
+            expect(reviews).toHaveLength(0);
+        })
+    });
+
+    test('200: should return all reviews sorted by review_id ascending', () => {
+        return request(app).get("/api/reviews?sort_by=review_id&order=asc")
+        .expect(200)
+        .then(({body})=>{
+            const reviews = body.reviews
+            expect(reviews).toHaveLength(13);
+            expect(reviews).toBeSortedBy("review_id")
+        })
+    });
+
+    test('200: should return all reviews sorted by review_image_url descending', () => {
```

Comment:
Please rename this one too. Pop me a message if you need me to clarify - the descriptions should refer to the general behaviour, not it's behaviour in a specific instance.